scripts/tools/install_stubs.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+LIBRARIES=(cloudpickle deepmerge fs huggingface_hub importlib_metadata joblib multipart pandas pathspec PIL starlette transformers xgboost yaml)
```

Comment:
should we have a requirements.txt file just for typing, that locks to a version of these libraries?

Reply:
sure we can do that

Reply:
done